library/src/main/kotlin/de/adesso/junitinsights/tools/ReportCreator.kt

```diff
@@ -27,25 +27,26 @@ object ReportCreator {
      */
     private fun processClassEvents(events: List<Event>): TestClass {
         val beforeAll = events[0].timeStamp.time
-        val spring = if (events[1].name.equals("context refreshed")) events[1].timeStamp.time else events[0].timeStamp.time
+        val spring = if (events[1].name == "context refreshed") events[1].timeStamp.time else events[0].timeStamp.time
         val afterAll = events.last().timeStamp.time
         val eventsGroupedByMethods = groupEventsByMethod(events)
         val methods = eventsGroupedByMethods.map { methodEvents -> processMethodEvents(methodEvents) }
 
         if (methods.isEmpty())
-            return TestClass(events.last().className, methods, 0, 0, 0, 0, 0, 0, 0)
+            return TestClass(events.last().className, events.first().timeStamp.time, methods, 0, 0, 0, 0, 0, 0, 0)
```

Comment:
Why don't you access the first element of the list by `events[0]`?

Reply:
I Like the more semantic version better, no other reason

Reply:
I think you should use the `events[0]` syntax here, because we used it everywhere else in the project.